src/style.css

```diff
@@ -1,3 +1,81 @@
-body {
+
+html {
+  font-family: Roboto, sans-serif;
+
+  --main-color: #00ACDC;
+}
+
+.nav-list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+.header .container {
+  display: flex;
+  justify-content: space-between;
+  align-items:center;
+  background-color: white;
+  padding: 0 50px;
+}
+
+.nav-list li {
+  display: inline-block;
+  padding: 0;
+  margin: 0;
+}
+
+.nav-logo {
   margin: 0;
+  padding: 0;
+  font-size: 0;
+}
+
+.nav-link {
+  text-decoration: none;
+  font-size: 12px;
+  font-style: normal;
+  text-transform: uppercase;
+  margin-right: 20px;
+  color: black;
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  position: relative;
+}
+
+.last {
+  margin-right: 0;
+}
+
+.nav-list li a:hover {
```

Comment:
It's better to use classes only for styling elements
Consider using tag selectors to add resets or page-wide styles